### PR TITLE
API and Resource implementation

### PR DESCRIPTION
**Pull request by eluciano11**

### CODE REVIEW

File: src/components/inscribete/FindVoterCenter/findYourCenterMachine.tsx [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
-const getVoterDetails = (voterId?: string) =>
-  fetch(`https://api.paravotar.org/consulta?voterId=${voterId}`).then(
-    response => {
-      if (!response.ok) {
-        throw new Error("HTTP status code: " + response.status)
-      }
-      return response.json()
-    }
-  )
+const getVoterDetails = async (voterId?: string) => {
```
Comment (Jpadilla1) [inline src/components/inscribete/FindVoterCenter/findYourCenterMachine.tsx]: Async/Await seems unnecessary here if we are going to return a promise already

File: src/packages/practica/resource.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import api from "../../services/api"
+import { BallotsResponse, OcrResult, VoterInfo } from "./services/types"
+import { CDN_URL } from "./services/constants"
+
+export const VoterInformationResource = {
+  async getVoterInfo(voterId?: string) {
```
Comment (Jpadilla1) [inline src/packages/practica/resource.ts]: same for all of these

**Review by Jpadilla1 — DISMISSED** [review]

File: src/components/inscribete/FindVoterCenter/findYourCenterMachine.tsx (hunk 1 above)
Comment (eluciano11) [reply src/components/inscribete/FindVoterCenter/findYourCenterMachine.tsx]: done.

File: src/packages/practica/resource.ts (hunk 2 above)
Comment (eluciano11) [reply src/packages/practica/resource.ts]: done.

**Review by Jpadilla1 — APPROVED** [review]

**Pull request merged**